io/post_fv3.F90

```diff
@@ -628,15 +627,15 @@ subroutine set_postvars_fv3(wrt_int_state,grid_id,mype,mpicomp)
       spval = 9.99e20
 !
 ! nems gfs has zhour defined
-      tprec   = float(wrt_int_state%fhzero)
+      tprec   = wrt_int_state%fhzero
       tclod   = tprec
       trdlw   = tprec
       trdsw   = tprec
       tsrfc   = tprec
       tmaxmin = tprec
       td3d    = tprec
-!      if(mype==0)print*,'MP_PHYSICS= ',imp_physics,'tprec=',tprec,'tclod=',tclod, &
-!       'dtp=',dtp,'tmaxmin=',tmaxmin,'jsta=',jsta,jend,im,jm
+      if(mype==0)print*,'MP_PHYSICS= ',imp_physics,'tprec=',tprec,'tclod=',tclod, &
+       'dtp=',dtp,'tmaxmin=',tmaxmin,'jsta=',jsta,jend,im,jm
```

Comment:
I am guessing this is probably a debugging print statement that you don't want to leave uncommented?

Reply:
Commented out.